pkg/v1/cli/component/prompt_test.go

```diff
@@ -66,3 +66,17 @@ func Test_translatePromptConfig_Input(t *testing.T) {
 	_, ok := prompt.(*survey.Input)
 	assert.True(ok)
 }
+
+func Test_PromptOptions(t *testing.T) {
+	assert := assert.New(t)
+
+	options := defaultPromptOptions()
+	opts := translatePromptOpts(options)
+
+	assert.NotNil(options)
+	assert.NotNil(opts)
+	assert.Equal("?", options.Icons.Question.Text)
+	assert.Equal("cyan+b", options.Icons.Question.Format)
+	assert.Equal(2, len(opts))
```

Comment:
I think it would be a good test to make sure that that `translatePromptOpts()` does not miss any fields.
I think you could create an empty `AskOptions` variable and apply both `opts` to it and then make sure the contents match what is expected.